src/lib/pages/interaction/component/drawer/selector/SelectorInput.tsx

```diff
@@ -38,7 +38,10 @@ export const ModuleSelectorInput = ({
 }: ModuleSelectorInputProps) => {
   const [keyword, setKeyword] = useState(selectedAddress.address as string);
   const [error, setError] = useState("");
-  const [addr, moduleName] = useMemo(() => keyword.split("::"), [keyword]);
+  const [addr, moduleName, functionName] = useMemo(
+    () => splitModule(keyword),
+    [keyword]
+  );
```

Comment:
Should the module or function name be in uppercase or start with a capital letter?

Reply:
I believe its all in lowercase